src/renderer/components/Header/ExecutionButtons.tsx

```diff
@@ -71,10 +71,9 @@ export const ExecutionButtons = memo(() => {
                 <IconButton
                     aria-label={t('header.stopButton', 'Stop button')}
                     colorScheme="red"
-                    disabled={
-                        !(status === ExecutionStatus.RUNNING || status === ExecutionStatus.PAUSED)
-                    }
+                    disabled={![ExecutionStatus.RUNNING, ExecutionStatus.PAUSED].includes(status)}
                     icon={<IoStop />}
+                    isLoading={[ExecutionStatus.KILLING].includes(status)}
```

Comment:
```suggestion
                    isLoading={ExecutionStatus.KILLING === status}
```

Reply:
whoops, leftover from when i had _another_ status in there as well